lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
What if `Cable.createConsumer` kept track of the cables you've set up? So if you try to connect to an already connected cable, it will return the existing connection instead? Should be fairly easy to key based on the url.

Can you forsee a situation when you would want multiple consumers for the same url?